content/blog/2022/07/verify-signatures-with-gha-and-prefix.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Verify image signatures with GithubActions and KeylessPrefix 
+authors:
+- Raul Cabello Martin 
+date: 2022-07-14
+---
+
+With the latest releases of Kubewarden v1.1.0 and
+the [verify-image-signatures](https://github.com/kubewarden/verify-image-signatures) policy, it's now possible to use
+`GithubActions` or `KeylessPrefix` for verifying images. Read our
+previous [blog post](https://www.kubewarden.io/blog/2022/05/verifying-image-signatures/) if you want to learn more about
+how to verify container images with Sigstore using Kubewarden.
+
+## Let's see it in action!
+
+We want to verify the image `ghcr.io/raulcabello/app-example` which was built and signed inside a GitHub action
+using this [GitHub Action](https://github.com/raulcabello/app-example/blob/master/.github/workflows/ci.yml).
+
+Out of the box, GitHub Actions have a specially crafted environment that makes Sigstore keyless signing work in
+a non-interactive way.
+The signatures produced in this way contain unique secure information that allow us to identify the GitHub owner
+(be it an individual or an organization) and the GitHub repository inside of which the GitHub Action has been
+executed.
+
+Starting from today, Kubewarden provides a convenient way to check signatures produced by GitHub actions.
+
+For this example, a Kubernetes cluster with Kubewarden already installed is required. The installation process is
+described in the [quick start guide](https://docs.kubewarden.io/quick-start).
+
+### GitHubAction
+
+```yaml 
+apiVersion: policies.kubewarden.io/v1
+kind: ClusterAdmissionPolicy
+metadata:
+  name: verify-image-signatures-policy
+spec:
+  module: registry://ghcr.io/kubewarden/policies/verify-image-signatures:v0.1.4
+  rules:
+  - apiGroups: [""]
+    apiVersions: ["v1"]
+    resources: ["pods"]
+    operations:
+    - CREATE
+    - UPDATE
+  mutating: true
+  settings:
+    signatures:
+      - image: "ghcr.io/raulcabello/app-example:*" # match all tags 
+        github_actions:
+        - owner: "raulcabello"
+          repo: "app-example" #optional. if omited all apps from the owner will ve valid.
+```
+
+This policy verifies all containers with an image that is `ghcr.io/raulcabello/app-example`. It will accept
+containers that were signed inside a GitHub action for the owner `raulcabello` and the app `app-example`.
```

Comment:
```suggestion
containers that were signed inside a GitHub Action job, run under the owner `raulcabello` and the repo `app-example`.
```
Or something similar, so people understand we are validating things signed in job runs.